cypress/e2e/read-receipts/redactions.spec.ts

```diff
@@ -263,10 +263,12 @@ describe("Read receipts", () => {
                 // Then the room is still read
                 assertRead(room2);
             });
-            it("Reacting to a redacted message leaves the room read", () => {
+            // Flakes because sometimes the unread count stays at 2
```

Comment:
should we have an issue for this?

Reply:
I'm going to handle flakes the same way I am handling failures, as part of https://github.com/vector-im/element-web/issues/24392 so I don't think we need one.